lib/api/README.md

```diff
@@ -1,7 +1,8 @@
 # `wasmer` [![Build Status](https://github.com/wasmerio/wasmer/workflows/build/badge.svg?style=flat-square)](https://github.com/wasmerio/wasmer/actions?query=workflow%3Abuild) [![Join Wasmer Slack](https://img.shields.io/static/v1?label=Slack&message=join%20chat&color=brighgreen&style=flat-square)](https://slack.wasmer.io) [![MIT License](https://img.shields.io/github/license/wasmerio/wasmer.svg?style=flat-square)](https://github.com/wasmerio/wasmer/blob/master/LICENSE) [![crates.io](https://img.shields.io/crates/v/wasmer.svg)](https://crates.io/crates/wasmer)
 
-[`Wasmer`](https://wasmer.io/) is the most popular [WebAssembly](https://webassembly.org/)
-runtime for Rust (...and also the fastest). It supports JIT (Just in Time) and AOT (Ahead of time)
+[`Wasmer`](https://wasmer.io/) is the most popular
+[WebAssembly](https://webassembly.org/) runtime for Rust (...and also
+the fastest). It supports JIT (Just in Time) and AOT (Ahead of time)
```

Comment:
Now that engines have more generic names, should we change this?

```suggestion
the fastest). It supports JIT (Just in Time, via the Universal engine) and AOT (Ahead of time, via the Native Shared Object engine)
```

Reply:
I think we should make a final pass on the `README.md` after all the renamings.